FBReader/src/main/java/org/geometerplus/android/fbreader/benetech/FBReaderWithNavigationBar.java

```diff
@@ -574,7 +574,11 @@ private String getNextParagraph() {
                 }
                 myCurrentSentence = currentSentence;
             }
-
+            try {
+                Thread.sleep(100);
+            } catch (InterruptedException e) {
+                e.printStackTrace();
```

Comment:
Can we log the exception instead of printing to console.  The try catch can be extracted into a method and the method name should have a name describing why we are sleeping.